internal/command/command_service.go

```diff
@@ -184,12 +178,12 @@ func (cs *CommandService) CancelSubscription(ctx context.Context) {
 }
 
 func (cs *CommandService) subscribe(ctx context.Context) {
-	commonSettings := &config.CommonSettings{
-		InitialInterval:     cs.agentConfig.Common.InitialInterval,
-		MaxInterval:         cs.agentConfig.Common.MaxInterval,
-		MaxElapsedTime:      createConnectionMaxElapsedTime,
-		RandomizationFactor: cs.agentConfig.Common.RandomizationFactor,
-		Multiplier:          cs.agentConfig.Common.Multiplier,
+	commonSettings := &config.BackOff{
+		InitialInterval:     cs.agentConfig.Client.Backoff.InitialInterval,
+		MaxInterval:         cs.agentConfig.Client.Backoff.MaxInterval,
+		MaxElapsedTime:      cs.agentConfig.Client.Backoff.MaxElapsedTime,
```

Comment:
For the `Subscribe` & `CreateConnection` RPCs, we always want to retry indefinitely. Otherwise if we stop retrying then the agent will just keep running doing nothing and never connect to the management plane.

Hence why the `MaxElapsedTime` is set to `0` for these RPCs.